services/web-app/components/page-nav/page-nav.js

```diff
@@ -93,11 +90,13 @@ const PageNav = ({ contentRef, items = [] }) => {
 
   const setSelectedItem = () => {
     const navLinks = Array.from(contentRef.current.querySelectorAll('[class^="page-nav_link"]'))
-    const sections = contentRef.current.querySelectorAll('[id]')
+    const sections = Array.from(contentRef.current.querySelectorAll('[id]'))?.filter((section) =>
+      items.some((item) => item.id === section.id)
+    )
 
-    let activeSession = null
+    let activeSection = null
     // Space between top of screen and where we want section to be "Active"
-    const scrollDistance = 90
+    const scrollDistance = 120
```

Comment:
Since this is used in more than one place in this file, it might be good to define it as a constant at the top.

Reply:
updated @jdharvey-ibm